### PR TITLE
Refactor and remove url upload function

### PR DESCRIPTION
**Pull request by risenW**
This PR removes the upload by URL feature from the Upload options

### CODE REVIEW

File: src/components/InputFile/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            <div className="upload-area__drop">
+                <input
+                    className="upload-area__drop__input"
+                    type="file"
+                    name="file"
+                    onChange={onChangeHandler}
+                />
+                <img
+                    className="upload-area__drop__icon"
+                    src="https://www.shareicon.net/data/256x256/2015/09/05/96087_cloud_512x512.png"
```
Comment (sglavoie) [inline src/components/InputFile/index.jsx:17]: I think it would be best to load this locally but that's fine for now I guess. :stuck_out_tongue:

**Review by sglavoie — APPROVED** [review]
LGTM :slightly_smiling_face:. 

(Sorry for the delay in reviewing :wink:)

File: src/components/InputFile/index.jsx (hunk 1 above)
Comment (risenW) [reply src/components/InputFile/index.jsx:17]: True, I'll update that.

**Review by sglavoie — APPROVED** [review]
Looks good! Thank you. :slightly_smiling_face:

**Pull request merged**